### PR TITLE
Spacy3 `en_core_sci_scibert`

### PR DESCRIPTION
**Pull request by DeNeutoy**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -37,6 +38,7 @@ Our models achieve performance within 3% of published state of the art dependenc
 | en_core_sci_sm | 89.75| 87.80  |  98.41  |  67.92  |  88.18  |
 | en_core_sci_md | 90.36| 88.53 |  98.49 |  68.58  |  88.27  |
 | en_core_sci_lg | 90.30| 88.48  |  98.53  |  69.21  |  88.67  |
+| en_core_sci_lg | 92.03| 90.25  |  98.91  |  67.91  |  92.21  |
```
Comment (danielkingai2) [inline docs/index.md:41]: typo on model here

File: project.yml [hunk 2]
```diff
@@ -540,4 +627,4 @@ commands:
 
 
 
-  # TODOs: evaluate everything, package command, another evaluate of the package, twiddle params, maybe add a command for uploading models and releasing?
\ No newline at end of file
+  # TODOs: evaluate everything, package command, another evaluate of the package, twiddle params, maybe add a command for uploading models and releasing?
```
Comment (danielkingai2) [inline project.yml:630]: can you delete this comment?

**Review by danielkingai2 — APPROVED** [review]
LGTM, minor comments

**Comment by SantoshGuptaML** [comment]
Has NER been added already? I see in the releases that NER is yet to be added

https://github.com/allenai/scispacy/releases

But it seems to run fine in the code

```
import spacy
nlp = spacy.load("en_core_sci_scibert")
doc2 = nlp("Spinal and bulbar muscular atrophy (SBMA) is an \
           inherited motor neuron disease caused by the expansion \
           of a polyglutamine tract within the androgen receptor (AR). \
           SBMA can be caused by this easily.")
doc2.ents

> (Spinal,
>  bulbar muscular atrophy,
>  SBMA,
>  inherited,
>  motor neuron disease,
>  expansion,
>  polyglutamine tract,
>  androgen receptor,
>  AR,
>  SBMA)
```

**Comment by danielkingai2** [comment]
NER is there, it is just not using scibert, but rather is using spacy's normal embedder.

**Comment by SantoshGuptaML** [comment]
ah I see. Is there an idea when Scibert Ner would be added?

**Comment by danielkingai2** [comment]
Not really. Maintaining scispacy is not my main project, so I don't have timelines for anything. It should't be hard to train your own if you want to. The whole training pipeline is in a config file, using spacy projects. Feel free to open an issue and ask questions if you are interested in doing this.

**Pull request merged**